examples/ftpd/ftpd_main.c

```diff
@@ -154,35 +154,75 @@ static void ftpd_accounts(FTPD_SESSION handle)
     }
 }
 
+static void hint(void)
+{
+  fprintf(stderr,
+      "Usage: ftpd [-46] \n\
+      \t-4    Use IPv4\n\
+      \t-6    Use IPv6\n\
+      ");
+}
+
 /****************************************************************************
  * Name: ftpd_daemon
  ****************************************************************************/
 
 int ftpd_daemon(int s_argc, char **s_argv)
 {
   FTPD_SESSION handle;
-  int ret;
+  int ret = EXIT_FAILURE;
+  int option;
+  int family;
+  bool badarg = false;
 
   /* The FTPD daemon has been started */
 
   g_ftpdglob.running = true;
   printf("FTP daemon [%d] started\n", g_ftpdglob.pid);
 
-  /* Open FTPD */
-
 #if ADDR_FAMILY == AF_INET6
-  handle = ftpd_open(CONFIG_EXAMPLES_FTPD_PORT, AF_INET6);
+  family = AF_INET6;
 #else
-  handle = ftpd_open(CONFIG_EXAMPLES_FTPD_PORT, AF_INET);
+  family = AF_INET;
 #endif
```

Comment:
Let's remove this and change variable declaration to `int family = ADDR_FAMILY;`

Reply:
Or to `int family = AF_UNSPEC;` like in `apps/examples/ftpc/ftpc_main.c`